astropy/visualization/wcsaxes/core.py

```diff
@@ -617,6 +617,8 @@ def get_tightbbox(self, renderer, *args, **kwargs):
         if not self.get_visible():
             return
 
+        # Do a draw to populate the self._bboxes list
+        self.draw_wcsaxes(renderer)
```

Comment:
How was this line not hit by a figure test? Are we not getting coverage out of circle?